doc/documents/Examples_Tutorials/Examples_Tutorials.rst

```diff
@@ -0,0 +1,623 @@
+   |                           |         num_output: 32 pad: 2 kernel_size: 5 stride: 1  }}    |
+   |                           |    layer {                                                    |
+   |                           |      name: "relu2"                                            |
+   |                           |      type: "ReLU"                                             |
+   |                           |      bottom: "conv2"                                          |
+   |                           |      top: "conv2"}                                            |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+
+   |                           | .. code:: c++                                                 |
+   |                           |                                                               |   
+   |                           |    mli_status mli_krn_conv2d_chw_fx8_k5x5_str1_krnpad(        |
+   |                           |      const mli_tensor * in,       // Input tensor             |
+   |                           |      const mli_tensor * weights,  // Weights tensor           |
+   |     **MLI Function**      |      const mli_tensor * bias,     // Biases tensor            |
+   |                           |      const mli_conv2d_cfg * cfg,  // Convolution config       |
+   |                           |      mli_tensor * out             // Output tensor            |
+   |                           |   );                                                          |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+
+   |                           | .. code:: c++                                                 |
+   |                           |                                                               |   
+   |                           |    mli_conv2d_cfg shared_conv_cfg = {                         |
+   |                           |      .stride_height = 1, .stride_width = 1,                   |
+   |                           |      .padding_bottom = 2, .padding_top = 2,                   |
+   | **MLI Function Config**   |      .padding_left = 2, .padding_right = 2,                   |
+   |                           |      .relu.type = MLI_RELU_GEN                                |
+   |                           |     };                                                        |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+ 
+
+..
+
+.. image::  ../images/5_op_map_step3.png
+   :align: center
+   :alt: CIFAR-10 Graph Visualization: Pooling Layers
+
+.. 
+
```

Comment:
Was "*MLI Pooling behavior differs from Caffe default one. In Caffe padding is implied for some combinations of layer parameters, even if not specified. We should indicate padding clearly because it is meant in the Caffe. It was done for compatibility with other frameworks.*"  part omitted intentionally here?